reloader/manifest.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: Reloader
```

Comment:
name: reloader
title: "Reloader"
version: "v0.0.91"

Can we have like this fo the manifest.yaml

Reply:
ok, will change :)